Sources/CardPayments/CardVaultDelegate.swift

```diff
@@ -17,4 +17,19 @@ public protocol CardVaultDelegate: AnyObject {
     ///   - client: the CardClient associated with delegate
     ///   - didFinishWithError: the error returned by the Card vault flow
     func card(_ cardClient: CardClient, didFinishWithVaultError vaultError: CoreSDKError)
+
+    /// Notify that the Card vault flow has been cancelled
+    /// - Parameters:
+    ///   - client: the CardClient associated with delegate
+    func cardVaultDidCancel(_ cardClient: CardClient)
```

Comment:
Is there a way to more similarly match the naming patterns of these new delagate methods to the ones above?

Reply:
This one was because the other delegate function, there were clear ways to distinguish different delegate functions with the parameter: func card(_ cardClient: CardClient, `didFinishWithVaultResult` vaultResult: CardVaultResult)
I wanted to distinguish cancel for checkout flow vs cancel for vaulting.

Reply:
Maybe `cardDidCancelVault()` ?

Reply:
I like that better. I was just referring to the flow CardVault, but it does read funny